src/index.html

```diff
@@ -11,7 +11,177 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="personalInformation">
+        <legend>Personal information</legend>
+        <label class="formField">
+          Surname:
+          <input
+            class="input"
+            type="text"
+            name="userSurname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="formField">
+          Name:
+          <input
+            class="input"
+            type="text"
+            name="userName"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="formField">
+          How old are You?
+          <input
+            class="input"
+            type="number"
+            name="userAge"
+            value="12" min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="formField">
+          Full date of birth:
+          <input
+            class="input"
+            type="date"
+            name="userBirthdate"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="formField">
+          I accept the term of the agreement
+          <input
+            class="input"
+            type="checkbox"
+            name="userAcception"
+            value="12" min="1"
```

Comment:
you should be consistent about spacing/moving to a new line. If you have 2+ attributes - you always move `each of them` to a new line
```suggestion
            value="12" 
            min="1"
```